.ci/ansible/inventory.yaml

```diff
@@ -3,7 +3,7 @@ all:
   children:
     containers:
       hosts:
-        pulp:
+        pulp.example.com:
```

Comment:
I fear this change will be able to make the CI green by hiding the original problem.
I tend to say we need a `OCI_REGISTRY_ORIGIN` setting.